main/src/ca/uwaterloo/flix/language/phase/Monomorpher.scala

```
@@ -114,6 +114,7 @@ object Monomorpher {
       case Kind.RecordRow => Type.RecordRowEmpty
       case Kind.SchemaRow => Type.SchemaRowEmpty
       case Kind.Predicate => Type.mkAnyType(tpe0.loc)
+      case Kind.JvmConstructorOrMethod => Type.mkAnyType(tpe0.loc) // TODO INTEROP I don't really know what to put at this stage.
```

This should throw an exception